File: app/src/main/java/com/github/kongpf8848/animation/fragment/TitleFragment.java
package com.github.kongpf8848.animation.fragment;

import android.widget.TextView;

import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.xsdk.ui.fragment.BaseFragment;
import com.github.kongpf8848.xsdk.ui.fragment.BaseLazyFragment;

import java.io.FileInputStream;
import java.io.FileOutputStream;
import java.io.IOException;
import java.io.InputStream;
import java.io.OutputStream;

import butterknife.BindView;

public class TitleFragment extends BaseLazyFragment {

    @BindView(R.id.tv_title)
    TextView tv_title;

    @Override
    public int getLayoutId() {
        return R.layout.fragment_title;
    }


    @Override
    protected void onFragmentFirstVisible() {
        super.onFragmentFirstVisible();
        tv_title.setText(getArguments().getString("title",""));

    }


}

File: app/src/main/java/com/github/kongpf8848/animation/fragment/TitleFragment.kt
package com.github.kongpf8848.animation.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.annotation.Nullable
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.base.BaseFragment

class TitleFragment : BaseFragment() {

    lateinit var tv_title:TextView
    companion object{
        fun getInstance(title:String):TitleFragment{
            return TitleFragment().apply {
                arguments = Bundle().apply {
                    putString("title", title)
                }
            }
        }
    }
    override fun onCreateView(inflater: LayoutInflater, @Nullable container: ViewGroup?, @Nullable savedInstanceState: Bundle?): View? {
       return inflater.inflate(R.layout.fragment_title,container,false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        tv_title=view.findViewById(R.id.tv_title)
        tv_title.text = arguments?.getString("title", "")
    }

}